.github/workflows/openhands-repair.yml

```
@@ -0,0 +1,142 @@
+
+          if [ "$COMMITS_SINCE_LABEL" -gt 0 ]; then
+            echo "has_changes=true" >> $GITHUB_OUTPUT
+            echo "OpenHands made $COMMITS_SINCE_LABEL commit(s)"
+          else
+            echo "has_changes=false" >> $GITHUB_OUTPUT
+            echo "No changes made by OpenHands"
+          fi
+
+      - name: Push repairs to PR branch
+        if: steps.verify.outputs.has_changes == 'true'
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)